client/src/components/AuthorizedRoute.js

```diff
@@ -1,12 +1,11 @@
-import React, { useContext } from 'react';
+import React from 'react';
 import { Navigate } from 'react-router';
-import { StateContext } from '../App';
 
 const AuthorizedRoute = (props) => {
   const { children } = props;
-  const { state } = useContext(StateContext);
+  const loginToken = sessionStorage.getItem('loginToken');
```

Comment:
so wrong ni siya, dapat naa gihapon nang `const { state } = useContext(StateContext);` kay magkuha ta dapat ug data sa context state sa app. ang dapat buhaton is sa `App.tsx` where didto na intialize ang `state` context, kay mag butang ka didto ug logic nga ang intial state sa `token` nga variable is ang nakuha nimo sa `sessionStorage` IF naa and undefined if wala

Reply:
resolved